packages/components/src/components/step/step.stories.ts

```diff
@@ -21,207 +33,181 @@ export default {
   ]),
   argTypes,
   parameters: { ...parameters },
-  decorators: [withActions] as any
+  decorators: [
+    withActions,
+    (story: any) =>
+      html`<style>
+          #story--components-sd-step--default--primary {
+            width: min-content;
+          }</style
+        >${story()}`
+  ] as unknown
 };
 
 /**
  * Default: This shows sd-step in its default state.
  */
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return html`${generateTemplate({ args })}`;
   }
 };
 
 /**
- * Use the orientation attribute to set the axis of a step.
+ * Use the `size` attribute to set the size of a step.
+ */
+
+export const Size = {
+  name: 'Size',
+  render: () => html`
+    <div class="flex gap-12 w-min">
+      <sd-step size="lg">
+        <span slot="label">Large</span>
+      </sd-step>
+      <sd-step size="sm">
+        <span slot="label">Small</span>
+      </sd-step>
+    </div>
+  `
+};
+
+/**
+ * Use the `orientation` attribute to set the axis of a step.
  */
 
 export const Orientation = {
-  parameters: { controls: { exclude: 'orientation' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'orientation' }
-      },
-      args
-    });
-  }
+  name: 'Orientation',
+  render: () => html`
+    <div class="flex items-center gap-24 w-min">
```

Comment:
Could you show also the tail for vertical, pls

![image](https://github.com/user-attachments/assets/9f32cd9f-42e9-465b-bf2e-3abe0a73f8e4)

Reply:
Updated